app/helpers/api/bapi/getCatalogueItem.test.js

```diff
@@ -0,0 +1,25 @@
+import { getData } from 'buying-catalogue-library';
+import { getCatalogueItem } from './getCatalogueItem';
+import { solutionsApiUrl } from '../../../config';
+import { logger } from '../../../logger';
+
+jest.mock('buying-catalogue-library');
+
+describe('getCatalogueItem', () => {
+  afterEach(() => {
+    getData.mockReset();
```

Comment:
Could make use of the line `jest.resetAllMocks();` rather than `getData.mockReset();` but very minor & might want to be specific here.

Reply:
good shout I'll use this whenever I need to do that from now on 👍